main/src/cgeo/geocaching/connector/trackable/GeokretyConnector.java

```diff
@@ -191,6 +217,41 @@ public String getTrackableCodeFromUrl(@NonNull final String url) {
     }
 
     @Override
+    @Nullable
+    public String getTrackableTrackingCodeFromUrl(@NonNull final String url) {
+        // http://geokrety.org/m/qr.php?nr=<TRACKING_CODE>
+        final String gkTrackingCode = StringUtils.substringAfterLast(url, "qr.php?nr=");
+        if (StringUtils.isAlphanumeric(gkTrackingCode)) {
+            return gkTrackingCode;
+        }
+        return null;
+    }
+
+    /**
+     * Lookup Trackable Geocode from Tracking Code.
+     *
+     * @param trackingCode
+     *          the Trackable Tracking Code to lookup
+     * @return
+     *          the Trackable Geocode
+     */
+    @Nullable
+    public static String getGeocodeFromTrackingCode(final String trackingCode) {
```

Comment:
@samueltardieu I've looked deeper into call stack of this function. Concerned `getGeocodeFromTrackingCode()` function and calling function `searchTrackable()` are running on network thread. 
Here is the call stack from the caller function. Do you think it's ok? Thanks
![capture du 2015-10-22 22 59 56](https://cloud.githubusercontent.com/assets/176794/10678326/fdd2fe5a-7911-11e5-9d5c-1c7f4e37feb8.png)